server/cli/launch.py

```diff
@@ -269,9 +266,6 @@ def launch(
     # create the server
     server: TestServer = TestServer(app_config)
 
-    if not app_config.server__app__verbose:
-        log.setLevel(ERROR)
-
```

Comment:
why was this line not working? Was it because it was being overridden by `configure_loader`

Reply:
yes, it was being overridden by `configure_logging`